<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1288</width>
    <height>516</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>9999999</width>
    <height>9999999</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="firstpage">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1281</width>
      <height>801</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="firstmenu">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>240</y>
        <width>151</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>login</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>240</y>
        <width>151</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>signup</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>50</y>
        <width>341</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Manage system</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="login">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>160</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>230</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loginbutton">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>340</y>
        <width>80</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>340</y>
        <width>80</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="loginname">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>160</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="loginpassword">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>230</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::EchoMode::Password</enum>
      </property>
      <property name="placeholderText">
       <string>请输入</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="signin">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>130</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>210</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>290</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Confirm password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="signinname">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>130</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="signinpassword1">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>210</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::EchoMode::Password</enum>
      </property>
      <property name="placeholderText">
       <string>请输入</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="signinpassword2">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>290</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::EchoMode::Password</enum>
      </property>
      <property name="placeholderText">
       <string>请输入</string>
      </property>
     </widget>
     <widget class="QPushButton" name="signinbutton">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>380</y>
        <width>80</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>380</y>
        <width>80</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="del">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <widget class="QLineEdit" name="del_line">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>180</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>请输入id</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>320</y>
        <width>80</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>320</y>
        <width>80</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>quit</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="display">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <widget class="QTableWidget" name="table">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>1281</width>
        <height>381</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="back">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>430</y>
        <width>80</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="search">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <widget class="QLineEdit" name="ages">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>140</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>age</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="names">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>100</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="heights">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>180</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>height</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="genders">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>300</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>gender</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ranks">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>220</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>rank</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="grades">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>100</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>grade</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="class_s">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>140</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>class_num</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="scores">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>180</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>score</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="salarys">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>260</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>salary</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ids">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>260</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>id</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="weights">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>220</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>weight</string>
      </property>
     </widget>
     <widget class="QLabel" name="labels">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>40</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QCheckBox" name="isteachers">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>300</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>是否是教师</string>
      </property>
     </widget>
     <widget class="QPushButton" name="searchbutts">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>380</y>
        <width>80</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QPushButton" name="quitbutts">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>380</y>
        <width>80</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Quit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>340</y>
        <width>281</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Tag：for gender：0 for famale，1 for male</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="add">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>9999999</height>
      </size>
     </property>
     <widget class="QCheckBox" name="isteacher">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>300</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>是否是教师</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="name">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>100</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="age">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>140</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>age</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="height">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>180</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>height</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="weight">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>220</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>weight</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>260</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>id</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="gender">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>300</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>gender</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="grade">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>100</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>grade</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="classnum">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>140</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>class_num</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="score">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>180</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>score</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="rank">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>220</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>rank</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="salary">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>260</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>salary</string>
      </property>
     </widget>
     <widget class="QPushButton" name="nextbutt">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>380</y>
        <width>80</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>next</string>
      </property>
     </widget>
     <widget class="QPushButton" name="finishbut">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>380</y>
        <width>80</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>finish</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>40</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>340</y>
        <width>281</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Tag：for gender：0 for famale，1 for male</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="menu">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>70</y>
        <width>211</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Manage menu</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="searchbutt">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>200</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
     <widget class="QPushButton" name="delbutt">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>290</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addbutt">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>290</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>录入</string>
      </property>
     </widget>
     <widget class="QPushButton" name="statbutt">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>200</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>统计</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1288</width>
     <height>38</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
